jdaviz/configs/imviz/plugins/parsers.py

```diff
@@ -87,6 +87,7 @@ def _parse_image(app, file_obj, data_label, show_in_viewer, ext=None):
     app.add_data(data, data_label)
     if show_in_viewer:
         app.add_data_to_viewer("viewer-1", data_label)
+        app.get_viewer("viewer-1").state.layers[0].percentile = 95
```

Comment:
This only works when image is loaded. But let's say if you load two and then hide the first one and show the second one with that right menu checkbox thing, it goes back to min/max default. Not sure how to set this as Imviz global default. Any ideas?